### PR TITLE
Create humans.txt

### LINKED ISSUE #124: Add `humans.txt`
There are a few 404 on ELK, so I guess people want this file :)

### PR DESCRIPTION
**Pull request by prayashm**
Closes #124

### CODE REVIEW

File: humans.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* TEAM */
+William Durand
+Twitter: @TailorDev
```
Comment (willdurand) [inline humans.txt]: Should be @couac

File: humans.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Github: https://github.com/willdurand
+
+Julien Maupetit
+Twitter: @jmaupetit
```
Comment (willdurand) [inline humans.txt]: Should be @julienmaupetit

**Comment by willdurand** [comment]
👍

**Comment by jmaupetit** [comment]
👍

**Pull request merged**